src/components/ItaliaTheme/View/BandoView/BandoView.jsx

```diff
@@ -24,7 +24,7 @@ import {
   richTextHasContent,
   SkipToMainContent,
 } from '@italia/components/ItaliaTheme/View';
-
+import Module from '@italia/components/ItaliaTheme/View/DocumentoView/Module';
```

Comment:
A questo punto il componente Module lo portiamo fuori dal documento, tipo in common?

Reply:
yessss